tests/test_md_cli.py

```diff
@@ -342,3 +342,32 @@ def test_invalid_config():
     )
     assert result.exit_code == 1
     assert isinstance(result.exception, ValueError)
+
+
+def test_struct_name(tmp_path):
+    """ "Test specifying the structure name."""
+    struct_name = "EXAMPLE"
+    struct_path = tmp_path / struct_name
+    stats_path = tmp_path / f"{struct_name}-nvt-T10.0-stats.dat"
+    traj_path = tmp_path / f"{struct_name}-nvt-T10.0-traj.xyz"
+    final_path = tmp_path / f"{struct_name}-nvt-T10.0-final.xyz"
+    result = runner.invoke(
+        app,
+        [
+            "md",
+            "--struct",
+            DATA_PATH / "NaCl.cif",
+            "--ensemble",
+            "nvt",
+            "--steps",
+            "2",
+            "--temp",
+            "10",
+            "--struct-name",
+            str(struct_name),
```

Comment:
Actually this needs to change?
```suggestion
            str(struct_path),
```